src/object.c

```diff
@@ -968,7 +968,7 @@ struct redisMemOverhead *getMemoryOverheadData(void) {
         listRewind(server.clients,&li);
         while((ln = listNext(&li))) {
             client *c = listNodeValue(ln);
-            if (c->flags & CLIENT_SLAVE)
+            if (c->flags & CLIENT_SLAVE && !(c->flags & CLIENT_MONITOR))
```

Comment:
Maybe add a separate mh->clients_monitor? :-) @soloestoy

Reply:
Yes, and PUB/SUB client, have submitted in another PR #4502, and need more optimization. 

This single fix should be merged before 5.0 rc1 I think.